### ISSUE TITLE
No visible focus

### ISSUE THREAD

**Issue by hannolans** [UniversalViewer/universalviewer#455]
Some elements don't get a visible focus when those elements receive tab focus with keyboard. Buttons, fields and more information-link. It would be better for accessibility when the focus is highlighted in all browsers.

wcag2: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html

**Comment by edsilv** [comment]
This can be improved in your theme: https://github.com/sul-dlss/universalviewer/issues/14
Although I think it makes sense to update the generic styles for this.

**Comment by hannolans** [comment]
To my opinion the generic style should be wcag2 compliant. Otherwise users have to tweak the software to make it accessible and not everyone is aware of this issue.

**Comment by mejackreed** [comment]
I agree @hannolans . We made several improvements to v3 that add some of this functionality. Also converting all button intention elements to `<buttons>`
